pkg/controller/kubelet-config/kubelet_config_features.go

```diff
@@ -196,7 +196,7 @@ func generateKubeConfigIgnFromFeatures(cc *mcfgv1.ControllerConfig, templatesDir
 	if err != nil {
 		return nil, err
 	}
-	if nodeConfig != nil {
+	if nodeConfig != nil && role == ctrlcommon.MachineConfigPoolWorker {
```

Comment:
I see in the commit message you said `Also added a minor fix in featuregate controller sync to apply the workerlatencyprofiles change only to the worker mcp`

but this is any node config isn't it? Including e.g. cgroups? Wouldn't this block master-feature-kubelet MC from updating any field?

Reply:
The function only updates the workerlatencyprofiles related change to the kubelet and not the cgroups change.

Reply:
ah ok I see all the other usages have this check :+1: